rclcpp/test/test_function_traits.cpp

```diff
@@ -15,6 +15,7 @@
 #include <gtest/gtest.h>
 
 #include <rclcpp/function_traits.hpp>
+#include <string>
```

Comment:
Actually, I had to put the headers in this order because `cpplint` if `<string>` was before `<rclcpp/function_traits.hpp>`:

```
/home/esteve/Projects/ros2/ros2/rclcpp/rclcpp/test/test_function_traits.cpp:18:  Found C system header after C++ system header. Should be: test_function_traits.h, c system, c++ system, other.  [build/include_order] [4]
```